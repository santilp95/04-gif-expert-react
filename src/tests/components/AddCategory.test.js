import React from 'react';
import'@testing-library/jest-dom';
const { shallow } = require("enzyme");
const { AddCategory } = require("../../components/AddCategory");

describe('Pruebas componentes <SddCatgerory />', () => {

    const setCategories = () =>{}

    test('debe mostrrar correctamente', () => {
        const wrapper = shallow(<AddCategory setCategories={setCategories}/>)

        expect(wrapper).toMatchSnapshot();
    });
    
});
