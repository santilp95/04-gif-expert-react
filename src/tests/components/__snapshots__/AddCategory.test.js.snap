// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas componentes <SddCatgerory /> debe mostrrar correctamente 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value="Texto"
  />
</form>
`;
